src/Controls/src/BindingSourceGen/BindingSourceGenerator.cs

```diff
@@ -171,7 +180,7 @@ static bool IsWritable(ISymbol? symbol)
 			{
 				IPropertySymbol propertySymbol => propertySymbol.SetMethod != null,
 				IFieldSymbol fieldSymbol => !fieldSymbol.IsReadOnly,
-				_ => false,
+				_ => true,
```

Comment:
Do we really need to change the default? I'd rather add more specific cases to the switch and keep the default `false`.

Reply:
Arrays e. g. `(string[])` have no symbol (`GetSymbolInfo` returns null). Maybe it is possible to somehow extract this info from `TypeInfo`, but this would need looking into.

For me default `true` makes sense as per my understanding stuff is writable by default. We can add `readonly` or other modifier to change that. But should we have no modifiers we should be able to write.

Reply:
OK. We can always revisit this later.